### DISCUSSION TITLE
Improvement to hotkeys for logitech keyboard

### DISCUSSION THREAD

**Discussion by StephaneTactill** [xanderfrangos/twinkle-tray#139] [Ideas]
Hi,

First of all, thanks for this brilliant, efficient, easy and straight-to-the-point app! I have used for some time other apps that are all ugly, featureless, subscription-based (really?), or all of this. Thanks !

Now to my suggestion.
I use a logitech MX KEYS keyboard, it has two brightness buttons on the F1 and F2 keys.
By default the keyboard is set so that these keys behave as brightness, and not F1/F2.

Twinkle does not detect these keys. I would love to be able to use these as hotkeys, and control twinkle from my keyboard.

All the best for the future.

**Comment by xanderfrangos** [comment]
Hi @StephaneTactill, 

Detecting the built-in brightness keys is unfortunately not possible. However, you could add a shortcut in the Logitech G Hub with the "Launch application" shortcut. 

![Screenshot 2021-03-04 095145](https://user-images.githubusercontent.com/33106561/110172378-0c1d1e00-7dcb-11eb-88a2-dd64f40d0575.jpg)

Someone had a similar request here: https://github.com/xanderfrangos/twinkle-tray/issues/101#issuecomment-757480996

You can find information about the command line arguments here: https://github.com/xanderfrangos/twinkle-tray#command-line-arguments

I hope that helps!

**Reply by logancharlot** [reply]
Hi, I tried to follow your instructions to change the shortcut in Logitech Options (which is similar to Logitech G Hub), but couldn't find the file path in Windows. `%LocalAppData%\Programs\twinkle-tray\Twinkle Tray.exe` doesn't exist in my installation. Is there another path ?

Thanks in advance and thank you for your amazing app!

**Comment by btTeddy** [comment]
There is a very simple solution.

Go to Logitech options, assign keyboard shortcut ctr+alt+shift+up to FN+F1 and ctr+alt+shift+down to FN+F2.

Then assign same hotkeys in Twinkle tray.

Job done, buttons work from now
